src/OrchardCore.Modules/OrchardCore.OpenId/AdminMenu.cs

```diff
@@ -71,6 +71,17 @@ public void BuildNavigation(string name, NavigationBuilder builder)
                                   .LocalNav());
                     });
                 }
+
+                if (features.Any(feature => feature.Id == OpenIdConstants.Features.Client))
```

Comment:
I missed this in the previous PR, but this is not how it should be done. A specific `Startup` class is preferred, which will register a custom `AdminMenu` class for each feature. There are examples in the source code, I can think of the SQL Queries for instance that does it. /cc @PinpointTownes

Reply:
I disagree. Having a menu class per feature sounds like a total overkill and a call for duplicate code (because well, you have to re-create the lambda hierarchy in every class).